src/components/Link.tsx

```diff
@@ -0,0 +1,25 @@
+import NextLink from 'next/link';
+import { ReactNode } from 'react';
+import { css } from '@emotion/react';
+
+type LinkProps = {
+  href: string;
+  children: ReactNode;
```

Comment:
Try using `ReactElement` here, so that you don't need to use the `<>` fragment below

Reply:
I think the Fragment isn't needed, actually, but using ReactElement doesn't work with or without a Fragment:
![image](https://user-images.githubusercontent.com/9061026/208831383-6ee3152d-a4a6-46b5-867e-fde09376faed.png)